networking/p2p/src/main/java/tech/pegasys/teku/networking/p2p/libp2p/PeerManager.java

```diff
@@ -53,6 +54,7 @@
       new ConcurrentHashMap<>();
   private final ReputationManager reputationManager;
   private final List<PeerHandler> peerHandlers;
+  private final LabelledGauge peersLabelledGauge;
```

Comment:
I don't think we need to store this as a field - can just be a local variable in the constructor (the metric system will hold a reference to it so it won't be garbage collected).